data/data/agent/files/usr/local/bin/pre-start-agent.sh.template

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+ImagesEnvFile=${1}/agent-images.env
+
+IMAGE=`grep "AGENT_IMAGE" ${ImagesEnvFile}  | cut -d'=' -f2`
```

Comment:
It seems like it would be simpler to put image_for in a separate script (like on the bootstrap node) and call it here.

Reply:
Changed to just call image_for in release-image.sh similar to bootstrap.